### PR TITLE
Add user to the base image

### PR DESCRIPTION
**Pull request by ilbambino**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -5,7 +5,9 @@ ARG SECRETARY_VERSION=0.9.1
 
 COPY FWDRootCA.pem /tmp/FWDRootCA.crt
 
-RUN apk add --no-cache ca-certificates openssl tzdata bash curl && \
+RUN addgroup -S fwd && \
+    adduser -u 999 -h /opt/nxt/ctrl -s /bin/false -D -H -G fwd fwd && \
```
Comment (ocraviotto) [inline Dockerfile]: Are we going to set `/opt/nxt/ctrl` as home for all projects? How about `/opt/fwd`?

Comment (ilbambino) [inline Dockerfile]: my intention was to remove the `-h /opt/...` altogether. I guess in the last copy-paste it went through. It is not mandatory to set a home for the user. Does it provide some benefit that I am not aware of?

Comment (ocraviotto) [inline Dockerfile]: Of course setting the home to another path would imply other changes in ctrl Dockerfile... so maybe we can leave it that way.

**Comment by ocraviotto** [comment]
:+1:

**Pull request merged**